### ISSUE TITLE
Scrobbles multiple times when pausing on Elisa

### ISSUE THREAD

**Issue by dylan10182000** [InputUsername/rescrobbled#82]
I use KDE's Elisa player to play my music and rescrobbled to scrobble my local music to last.fm, but every time i pause and then play a song, it'll scrobble the track again.

**Comment by InputUsername** [comment]
What `min-play-time` are you using?

**Comment by dylan10182000** [comment]
I dont have it set. Is that the issue?

**Comment by InputUsername** [comment]
>I dont have it set. Is that the issue?

No, I was just wondering whether you had it set to some low value which means the song gets scrobbled again very quickly, but if it's not set it *should* wait until half the song has played.

Someone else had a similar issue (#78) but that turned out to be a problem with their music player. Is there any chance the same is happening in your case? If not, I'm not quite sure what's going wrong, but I'm happy to figure out what the issue is.

**Comment by gyro-cat** [comment]
The same is happening to me. I notice this is happening mostly with long songs. If the default minimum scrobble length is like listenbrainz's, (aka half the song or 4 minutes, whichever is shorter), I could in theory:

1. Have a song that is eg. 18 minutes long
2. I listen for the first 4 minutes, the song is scrobbled
3. Pause the song and keep listening from the same point later
4. 4 more minutes pass, the song is scrobbled again

I came to this conclusion checking the logs after playing a song then pausing it:
```
dic 18 00:06:22 fedora rescrobbled[1502]: Found active player Cantata
dic 18 00:06:22 fedora rescrobbled[1502]: ----
dic 18 00:06:22 fedora rescrobbled[1502]: Now playing: abriction - Her kiss in red light (Interstates)
dic 18 00:06:22 fedora rescrobbled[1502]: Status updated on ListenBrainz successfully
dic 18 00:06:23 fedora rescrobbled[1502]: ----
dic 18 00:06:23 fedora rescrobbled[1502]: Player Cantata stopped, looking for a new MPRIS player...
```

As you can see from the timestamp, immediately after pausing the daemon treats the player as inactive. If I now play the song again, even from the same point, it gets treated as a new play, which then could lead to new scrobbles.

**Comment by gyro-cat** [comment]
This actually creates another problem with shorter songs if they're paused more than once. If I'm listening to a 2 minute long song, pause at 0:50, resume, pause again at 1:20, resume and finish the song, nothing is scrobbled since consecutive play time never passed one minute.

**Comment by mloujapero726381837** [comment]
Can replicate in [kew](https://codeberg.org/ravachol/kew).
